src/sentry/event_manager.py

```diff
@@ -717,6 +717,8 @@ def _get_event_instance(self, project_id=None):
         # unused
         time_spent = data.pop('time_spent', None)
 
+        data['node_id'] = Event.get_node_id(project_id, event_id)
```

Comment:
If this is deterministic, is there a reason we need to send it along?

Reply:
Because here in `event_manager` we strip out all the data we would need to figure out the key. So when we pass down this data blob to the `NodeField`/`NodeData`, it doesn't have the `project_id` or the `event_id` so has no way to calculate the `node_id` I also didn't want to have the NodeField try and peek into the `Event` model to get these values, as a Field really shouldn't have a dependency on its Model. So the only way I could think of was to put the `node_id` into the data blob here.